### ISSUE TITLE
tslint doesn't detect missing comma in objects

### ISSUE THREAD

**Issue by timhwang21** [palantir/tslint#4070]
### Bug Report

- __TSLint version__: `"5.11.0"`
- __TypeScript version__: `"^2.9.2"`
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
const obj = {
	a: 1,
	b: 2
	c: 3,
};
```

with `tslint.json` configuration:

```json
{
  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
  "linterOptions": {
    "exclude": [
      "config/**/*.js",
      "node_modules/**/*.ts"
    ]
  }
}
```

#### Actual behavior

`tslint` does not detect the syntax error 

#### Expected behavior

`tslint` sees that there's a syntax error and warns.

I noticed `eslint` detects this error fine. I'm new to the Typescript ecosystem; is it expected that another linter or tool should be used in coordination with `tslint`? 

My primary concern is that I'm working on a project that relies on `tslint` in CI. Is there a more idiomatic way to set up a CI pipeline such that such errors are caught? Thanks!

**Comment by johnwiseheart** [comment]
My mistake - typescript already catches this, so `tslint` does not need to.

**Comment by timhwang21** [comment]
@johnwiseheart thanks for responding. Would you recommend running `tsc` in CI? Thank you!

**Comment by johnwiseheart** [comment]
Sure, surely you would be running `tsc` to generate some type of `js` bundle to use anyway right? I can't see any disadvantages to using `tsc` first, and then `tslint` after in your CI.

**Comment by timhwang21** [comment]
For better or worse we currently only do so when deploying (project is still in its early stages), and we want to have the peace of mind on every merge, not on every release. Again I'm fairly new to typescript so apologies if that was a silly question. Thanks!
